game.py

```diff
@@ -20,7 +20,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_count = 0
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word and not correct_letter_guess_statuses[user_input]:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[user_input] = True
+        else:
+            print(f"The letter {user_input} is not in the word.")
+            wrong_guesses_list.append(user_input)
+            wrong_guesses_count += 1
+        
+        print_snowman_graphic(wrong_guesses_count)
+        print(f"Wrong guesses: {wrong_guesses_list}")
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        if is_word_guessed(snowman_word,correct_letter_guess_statuses):
```

Comment:
Consider adding a few blank lines within your function to separate logically distinct sections of the code (e.g., setup, processing, return).  According to [PEP 8](https://peps.python.org/pep-0008/#blank-lines), blank lines can improve readability by visually grouping related operations. Right now, everything runs together, which can make it harder to quickly understand the structure and flow of your function.
```suggestion
        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
        
        if is_word_guessed(snowman_word,correct_letter_guess_statuses):
```